### DISCUSSION TITLE
Configuration  Nginx within Docker

### DISCUSSION THREAD

**Discussion by lehnbe** [netbox-community/netbox#18060] [Help Wanted!]
Hello I have setup Netbox from
https://github.com/netbox-community/netbox-docker
I found a notice that this runs nginx within the main netbox docker container.

I want to Access Netbox on a different port. I did not find the nginx.conf file. I also wasn't able to find instructions how to cusomize nginx within Docker. The regular instruxctions are all pointing to the nginx conf

**Comment by candlerb** [comment]
> I want to Access Netbox on a different port

Just like any other docker container, the mapping of external ports to ports in the container is a function of the container platform; you don't need to modify the container itself. For example, if you're using docker compose then this would be defined in the `docker-compose.yml` file.

Please see docker documentation for how to do this. e.g.
https://docs.docker.com/compose/how-tos/networking/
Look at the "ports:" section of the YAML.
